api/framer-motion.api.md

```diff
@@ -731,6 +732,10 @@ export type Variants = {
 };
 
 
+// Warnings were encountered during analysis:
+// 
+// types/motion/index.d.ts:217:5 - (ae-forgotten-export) The symbol "custom" needs to be exported by the entry point index.d.ts
+

```

Comment:
I think this warning is valid. It will work in the test, but when used imported from motion, it will just have type any